doc/setup/library_tests/index.md

```diff
@@ -0,0 +1,93 @@
+# Setting up library tests {#setup_tests}
+
+<b>Learning Objective:</b><br>
+In this guide you will learn to set up SeqAn3's library tests to make sure that your contributions don't break anything.
+
+\tutorial_head{Easy, 20 Minutes, \ref setup, }
+
+[TOC]
+
+# Unit testing
+
+Unit tests are the most important tests, they should cover all functionality and all code-paths of the library.
+Before submitting a pull request to our repository, make sure that the unit tests pass on your system.
+
+## Setting up unit tests
+
+Assume that you have cloned SeqAn including submodules into `/home/me/devel/seqan3` and performed some local changes.
+
+Create an out-of-source build directory and change to it:
+
+```bash
+mkdir -p /home/me/devel/seqan3-build/debug
+cd /home/me/devel/seqan3-build/debug
+```
+
+Invoke CMake (this is often referred to as the "Configure" step):
+
+```bash
+cmake ../../seqan3/test/unit -DCMAKE_BUILD_TYPE=Debug -DCMAKE_CXX_COMPILER=g++-7
+```
+
+The build type could be "Release", but stick to "Debug" for now. Specifying the compiler is optional; depending on
+your setup, you may need to give the full path.
+
+Invoke Make (this is the actual build step):
```

Comment:
Should we also talk about how to build a single test instead building the complete suite? :thinking:

Reply:
Good point!